src/components/screen/ProfileUpdate.tsx

```diff
@@ -89,7 +93,51 @@ function Screen(props: Props): React.ReactElement {
     }
   };
 
-  const onPressImg = (): void => {};
+    const getPermissions = async (
```

Comment:
Better naming here could be `requestPermissions`. `method` naming feels like if you are checking current permissions status to the system without asking any.

More likely, you can add `requestPermissions` method some where in `utils` so this could be called in other screens.